### PR TITLE
Styling for Forum

### PR DESCRIPTION
**Pull request by rmr167**
This includes the styling for the forum page as outlined in the capstone project log for Friday, August 28th.

### CODE REVIEW

File: src/main/webapp/forum.html [hunk 1]
```diff
@@ -3,9 +3,9 @@
   <head>
     <meta charset="UTF-8">
     <title>Seniors Connect</title>
+    <script src="https://kit.fontawesome.com/888d0bd439.js" crossorigin="anonymous"></script>
```
Comment (praneetmek) [inline src/main/webapp/forum.html:6]: what does this import statement do?

File: src/main/webapp/styles/general.css [hunk 2]
```diff
@@ -4,6 +4,10 @@
   --backColor: white;
   --mainColor: #4285f4;
   --placeholder: #b7b7b7;
+  --fGray: #f0f0ec;
+  --gRed: #db4437;
```
Comment (praneetmek) [inline src/main/webapp/styles/general.css:8]: what does the "f" "g" and "gL" mean?

Comment (nihabhaskar) [reply src/main/webapp/styles/general.css:8]: i made the var gRed bc its the google red

Comment (rmr167) [reply src/main/webapp/styles/general.css:8]: f is for forum

File: src/main/webapp/forum.html (hunk 1 above)
Comment (rmr167) [reply src/main/webapp/forum.html:6]: For the icons (thumbs-up and search)

**Pull request merged**